editors/arb-ltip/util.ts

```diff
@@ -0,0 +1,17 @@
+import { Maybe } from 'document-model/document-model';
+
+export function maybeToArray<T>(value: Maybe<Maybe<T>[]>): T[] {
+    if (!value) {
+        return [];
+    }
+
+    return value.filter(v => v !== null) as T[];
+}
+
+export function classNames(...classes: string[]) {
```

Comment:
No need to change but just fyi — we are using the [tailwind-merge](https://github.com/dcastil/tailwind-merge/blob/v2.2.2/docs/when-and-how-to-use-it.md) library which also has a `twJoin` function that does the same thing as what's defined here with a few quality of life helpers